IntelliTect.TestTools.Selenate/IntelliTect.TestTools.Selenate/IntelliTect.TestTools.Selenate.csproj

```diff
@@ -5,11 +5,15 @@
 
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
 
-    <Version>1.2.2</Version>
+    <Version>1.3.0-2019.20.08-alpha</Version>
```

Comment:
It is not already and I'd love to do that. :) It's on my to-do list, I just keep pushing it off. I think I honestly just need to finish Kyle's work that he started. I'll take a peek at the coding standards repo to see if it lines up with what I was thinking, and see if it can work for this. Standby!

Reply:
Cool. I don't think we have documented the setup for pipeline stuff, but if you have questions or need help let me know.

Reply:
Sounds good. I'll make an effort to tackle this next.